spec/rubocop/cop/offense_spec.rb

```diff
@@ -180,4 +180,49 @@ def Foo
       expect(offense.highlighted_area.source).to eq('Foo')
     end
   end
+
+  context 'when the location is pseudo' do
+    let(:location) { described_class::NO_LOCATION }
+
+    it 'returns a location with valid size and length' do
+      expect(offense.location.size).to eq 1
```

Comment:
I don't remember at all why `end_pos` is 1. Shouldn't it be 0 (and size too)?

Reply:
The following PR seems the origin of `end_pos=1`. I cannot find the reason. 😓 

https://github.com/rubocop-hq/rubocop/pull/2516/files#diff-4f6676361bec108acba5cfd5a6d979b3

<img width="711" alt="image" src="https://user-images.githubusercontent.com/473530/94873217-90205200-0489-11eb-8106-8205540fcf84.png">

Reply:
If you could try with 0, I think it would make more sense (especially on an empty file!)

Reply:
I try it via 7e3b3da.